invite/templates/invite/invite.html

```diff
@@ -19,7 +19,12 @@
 
             if (formCount > 1) {
                 // Delete the item/form
-                $(btn).parents('.control-group').remove();
+                var rowToRemove = $(btn).parents('.control-group')
+                // Delete any errors that might be associated with the row
+                while (rowToRemove.prev().hasClass('alert-error')) {
```

Comment:
You could do something like below instead of using a while statement. `prevUntil()` will include any elements that match the selector `:not(.alert)` and then filter to only include `.alert`. `.end()` returns the context of the chain to the previous selector `rowToRemove`.

Certainly no performance difference in such a small context, but just a little jQuery FYI.

```
rowToRemove.prevUntil(':not(.alert)', '.alert')
    .remove()
  .end()
    .remove() 
```

Reply:
I definitely wasn't thrilled about the `while` loop. I'll do as you suggest

Reply:
My code worked when I tried it in the JS console, but not positive it's exactly perfect for the function. Would only need minor tweaking if anything.

Reply:
understood, I'll tweak if necessary, it's definitely enough for me to work from

Reply:
realized I didn't even need the loop in the first place. there's only ever going to be one such element to delete